src/assets/img/startup-co-single.svg

```diff
@@ -0,0 +1,4 @@
+<svg width="48" height="49" viewBox="0 0 48 49" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Please use https://jakearchibald.github.io/svgomg/ or any other tool to minify svg icons.